doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -677,7 +677,13 @@ Modem libraries
 * :ref:`lib_location` library:
 
   * Added support for accessing nRF Cloud services using CoAP through the :ref:`lib_nrf_cloud_coap` library.
-  * Updated the neighbor cell search to use GCI search depending on :c:member:`location_cellular_config.cell_count` value.
+
+  * Updated:
+
+    * The neighbor cell search use GCI search depending on :c:member:`location_cellular_config.cell_count` value.
+    * The semantics of cellular and Wi-Fi timeouts only apply to neighbor cell measurement and Wi-Fi scan, respectively.
+      Earlier these timeouts applied to the upcoming cloud connection to send the data to the cloud for position resolution.
```

Comment:
```suggestion
      Earlier, these timeouts applied to the cloud connection to send the data to the cloud for position resolution.
```
Not sure I understood this correctly.

Reply:
We need some wording to tell that the timeouts used to apply to both neighbor cell search and the cloud connection or data transfer that was used to send the neighbor data to the cloud.

By removing "upcoming", this is either not the case or at least very unclear.

Reply:
Roger that. I had a bit of hard time digesting this.

Reply:
Comma added and "also" too. Please check.